dronekit/__init__.py

```diff
@@ -2748,6 +2794,65 @@ def __setitem__(self, index, value):
         self._vehicle._wpts_dirty = True
 
 
+class ServoOutputRaw(object):
+    """
+    The RAW servo readings from the OUTPUT of the autopilot
+    This contains the true raw values without any scaling to allow data capture and system debugging.
+
+    The message definition is here: https://pixhawk.ethz.ch/mavlink/#SERVO_OUTPUT_RAW
+
+
+    """
+
+    def __init__(self,
```

Comment:
this is quite the signature, did you explore any other options to the servo attributes?, perhaps kwargs?